vulnerabilities/migrations/0002_auto_20210105_1725.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.0.7 on 2021-01-05 17:25
```

Comment:
Could you squash this into one migration file ? 

To do this you would need to delete existing migrations and generate a migration script